osu.Game/Storyboards/Drawables/DrawableStoryboardSample.cs

```diff
@@ -21,6 +21,12 @@ public class DrawableStoryboardSample : PausableSkinnableSound
 
         public override bool RemoveWhenNotAlive => false;
 
+        /// <remarks>
+        /// Contrary to <see cref="PausableSkinnableSound"/>, all <see cref="DrawableStoryboardSample"/>s are affected
+        /// by sample disables, as they are oftentimes longer-running sound effects. This also matches stable behaviour.
+        /// </remarks>
+        protected override bool AffectedBySamplePlaybackDisable => true;
```

Comment:
My brain cannot comprehend this addition. Nothing really more to say, but I think this needs to be done in a different way.

Reply:
To clarify:

- what is a "sample disable"?
- The reasoning here doesn't really make sense. No mention of stable is required.

Proposal:

- Rename this property to `IsAffectedByISamplePlaybackDiasbler` or something
- Update the comment to read more like "Unlike the more general case of gameplay sounds (which are allowed to play out after a player pause, unless looping), storyboard samples could run for a longer duration. For this reason we opt to pause them in all cases."